pkg/scheduler/core/generic_scheduler.go

```diff
@@ -353,8 +353,7 @@ func (g *genericScheduler) findNodesThatPassExtenders(pod *v1.Pod, feasibleNodes
 		feasibleList, failedMap, err := extender.Filter(pod, feasibleNodes)
 		if err != nil {
 			if extender.IsIgnorable() {
-				klog.Warningf("Skipping extender %v as it returned error %v and has ignorable flag set",
-					extender, err)
+				klog.ErrorS(err, "Skipping extender as it returned error and has ignorable flag set", "extender", extender)
```

Comment:
Why change warning to error?

Reply:
see: https://github.com/kubernetes/enhancements/issues/1602#issuecomment-717101418 
and MR https://github.com/kubernetes/klog/pull/184

Reply:
I am not sure if changing to an `ErrorS` is better than simply using an `InfoS`. The enhancement comment you linked describes that the current semantics define low-verbosity Info messages as a similar intent to Warning messages.

Since this is currently a skippable Warning that doesn't affect running of the program, I don't think this should be an error.

Reply:
+1 to using an info here